src/test/resources/application-unittest.yaml

```diff
@@ -0,0 +1,78 @@
+spring:
```

Comment:
You need to rename the file name. I understand that we already have application-test.yaml and there's probably a clash, but you need to rename the integration test yaml file to application-integration.yaml and have application-test.yaml in test module. And of course you need to rename @ActiveProfiles("unittest") to @ActiveProfiles("test")

Reply:
This has been changed as asked